assets/js/steps/login-components/scan-password-login.js

```diff
@@ -54,13 +55,13 @@ function ScanPasswordLogin({ actionHandler }) {
     useEffect(() => {
         const barcodeScanner = new BarcodeScanner(BARCODE_SCANNING_TIMEOUT);
 
-        const barcodeCallback = (code) => {
-            if (code.length === BARCODE_COMMAND_LENGTH) {
-                if (code === BARCODE_COMMAND_FINISH) {
+        const barcodeCallback = (result) => {
+            if (result.type === BARCODE_TYPE_COMMAND) {
```

Comment:
I see this callback function as the same in all the different files?

Reply:
The accepted commands are different for each view. But it might be refactored... I will do that.